Rudder-Appsflyer/Classes/RudderAppsflyerIntegration.m

```diff
@@ -102,6 +102,9 @@ - (void) processRudderEvent: (nonnull RSMessage *) message {
                     }
                 }
             }
+            else {
+                afEventName = [afEventName stringByReplacingOccurrencesOfString:@" " withString:@"_"];
```

Comment:
Don't we need to lower case?

Reply:
Please refer [here](https://github.com/rudderlabs/rudder-integration-appsflyer-ios/blob/94de8d515b895080007d7782bb0349c84555f457/Rudder-Appsflyer/Classes/RudderAppsflyerIntegration.m#L61), we're already doing it.

Reply:
Cool.